app/src/utils/Fips.ts

```diff
@@ -58,61 +110,3158 @@ export const STATE_FIPS_MAP: Record<string, string> = {
+  "06013": "Contra Costa",
+  "06015": "Del Norte",
+  "06017": "El Dorado",
+  "06019": "Fresno",
+  "06021": "Glenn",
+  "06023": "Humboldt",
+  "06025": "Imperial",
+  "06027": "Inyo",
+  "06029": "Kern",
+  "06031": "Kings",
+  "06033": "Lake",
+  "06035": "Lassen",
+  "06037": "Los Angeles",
+  "06039": "Madera",
+  "06041": "Marin",
+  "06043": "Mariposa",
+  "06045": "Mendocino",
+  "06047": "Merced",
+  "06049": "Modoc",
+  "06051": "Mono",
+  "06053": "Monterey",
+  "06055": "Napa",
+  "06057": "Nevada",
+  "06059": "Orange",
+  "06061": "Placer",
+  "06063": "Plumas",
+  "06065": "Riverside",
+  "06067": "Sacramento",
+  "06069": "San Benito",
+  "06071": "San Bernardino",
+  "06073": "San Diego",
+  "06075": "San Francisco",
+  "06077": "San Joaquin",
+  "06079": "San Luis Obispo",
+  "06081": "San Mateo",
+  "06083": "Santa Barbara",
+  "06085": "Santa Clara",
+  "06087": "Santa Cruz",
+  "06089": "Shasta",
+  "06091": "Sierra",
+  "06093": "Siskiyou",
+  "06095": "Solano",
+  "06097": "Sonoma",
+  "06099": "Stanislaus",
+  "06101": "Sutter",
+  "06103": "Tehama",
+  "06105": "Trinity",
+  "06107": "Tulare",
+  "06109": "Tuolumne",
+  "06111": "Ventura",
+  "06113": "Yolo",
+  "06115": "Yuba",
+  "08001": "Adams",
+  "08003": "Alamosa",
+  "08005": "Arapahoe",
+  "08007": "Archuleta",
+  "08009": "Baca",
+  "08011": "Bent",
+  "08013": "Boulder",
+  "08014": "Broomfield",
+  "08015": "Chaffee",
+  "08017": "Cheyenne",
+  "08019": "Clear Creek",
+  "08021": "Conejos",
+  "08023": "Costilla",
+  "08025": "Crowley",
+  "08027": "Custer",
+  "08029": "Delta",
+  "08031": "Denver",
+  "08033": "Dolores",
+  "08035": "Douglas",
+  "08037": "Eagle",
+  "08039": "Elbert",
+  "08041": "El Paso",
+  "08043": "Fremont",
+  "08045": "Garfield",
+  "08047": "Gilpin",
+  "08049": "Grand",
+  "08051": "Gunnison",
+  "08053": "Hinsdale",
+  "08055": "Huerfano",
+  "08057": "Jackson",
+  "08059": "Jefferson",
+  "08061": "Kiowa",
+  "08063": "Kit Carson",
+  "08065": "Lake",
+  "08067": "La Plata",
+  "08069": "Larimer",
+  "08071": "Las Animas",
+  "08073": "Lincoln",
+  "08075": "Logan",
+  "08077": "Mesa",
+  "08079": "Mineral",
+  "08081": "Moffat",
+  "08083": "Montezuma",
+  "08085": "Montrose",
+  "08087": "Morgan",
+  "08089": "Otero",
+  "08091": "Ouray",
+  "08093": "Park",
+  "08095": "Phillips",
+  "08097": "Pitkin",
+  "08099": "Prowers",
+  "08101": "Pueblo",
+  "08103": "Rio Blanco",
+  "08105": "Rio Grande",
+  "08107": "Routt",
+  "08109": "Saguache",
+  "08111": "San Juan",
+  "08113": "San Miguel",
+  "08115": "Sedgwick",
+  "08117": "Summit",
+  "08119": "Teller",
+  "08121": "Washington",
+  "08123": "Weld",
+  "08125": "Yuma",
+  "09001": "Fairfield",
+  "09003": "Hartford",
+  "09005": "Litchfield",
+  "09007": "Middlesex",
+  "09009": "New Haven",
+  "09011": "New London",
+  "09013": "Tolland",
+  "09015": "Windham",
+  "10001": "Kent",
+  "10003": "New Castle",
+  "10005": "Sussex",
+  "11001": "District of Columbia",
```

Comment:
Can we move this to the state level? It technically can go in either but I think it's more obvious at the state level